crates/router/src/connector/netcetera/transformers.rs

```diff
@@ -552,6 +554,22 @@ impl TryFrom<&NetceteraRouterData<&types::authentication::ConnectorAuthenticatio
         };
         let browser_information = request.browser_details.map(netcetera_types::Browser::from);
         let sdk_information = request.sdk_information.map(netcetera_types::Sdk::from);
+        let device_render_options = match request.device_channel {
+            api_models::payments::DeviceChannel::App => {
+                Some(netcetera_types::DeviceRenderingOptionsSupported {
+                    // hard-coded until core provides these values.
+                    sdk_interface: netcetera_types::SdkInterface::Both,
```

Comment:
This should always be `Native` for us

Reply:
As per netcetera we are supposed to send this value.